src/support/slack/commands.js

```diff
@@ -42,6 +43,7 @@ export default (context) => [
   martechImpact(context),
   runAudit(context),
   runImport(context),
+  RunInternalReport(context),
```

Comment:
make the first letter lowercase so as to not confuse it with a class or functional object